Description: Recommend apt instead of apt-get
 It's cooler
Author: Julian Andres Klode <jak@debian.org>

--- command-not-found-0.2.38.orig/CommandNotFound/CommandNotFound.py
+++ command-not-found-0.2.38/CommandNotFound/CommandNotFound.py
@@ -186,9 +186,9 @@ class CommandNotFound:
                     print >>sys.stderr, _("This is most likely caused by the lack of administrative priviledges associated with your user account.")
             return False
 
-        # do not give advice if we are in a situation where apt-get
+        # do not give advice if we are in a situation where apt
         # or aptitude are not available (LP: #394843)
-        if not (os.path.exists("/usr/bin/apt-get") or
+        if not (os.path.exists("/usr/bin/apt") or
                 os.path.exists("/usr/bin/aptitude")):
             return False
 
@@ -201,10 +201,10 @@ class CommandNotFound:
             print >>sys.stderr, _("The program '%s' is currently not installed. ") % command,
             if posix.geteuid() == 0:
                 print >>sys.stderr, _("You can install it by typing:")
-                print >>sys.stderr, "apt-get install %s" %  packages[0][0]
+                print >>sys.stderr, "apt install %s" %  packages[0][0]
             elif self.user_can_sudo:
                 print >>sys.stderr, _("You can install it by typing:")
-                print >>sys.stderr, "sudo apt-get install %s" %  packages[0][0]
+                print >>sys.stderr, "sudo apt install %s" %  packages[0][0]
             else:
                 print >>sys.stderr, _("To run '%(command)s' please ask your administrator to install the package '%(package)s'") % {'command': command, 'package': packages[0][0]}
         elif len(packages) > 1:
@@ -213,9 +213,9 @@ class CommandNotFound:
             for package in packages:
                 print >>sys.stderr, " * %s" % package[0]
             if posix.geteuid() == 0:
-                print >>sys.stderr, _("Try: %s <selected package>") % "apt-get install"
+                print >>sys.stderr, _("Try: %s <selected package>") % "apt install"
             elif self.user_can_sudo:
-                print >>sys.stderr, _("Try: %s <selected package>") % "sudo apt-get install"
+                print >>sys.stderr, _("Try: %s <selected package>") % "sudo apt install"
             else:
                 print >>sys.stderr, _("Ask your administrator to install one of them")
         return len(packages) > 0
